test/scripts/ssh-setup.sh

```diff
@@ -128,6 +128,9 @@ function install_packages_apt {
     echo "Installing required apt packages"
     apt update
     apt install -yf xvfb wireguard-tools curl
+    if ! which ping &>/dev/null; then
+        apt install -yf iputils-ping || true
```

Comment:
Why not fail if ping fails to install? Won't some tests end up failing if that's the case?